src/cascadia/TerminalApp/TabHeaderControl.xaml

```diff
@@ -11,6 +11,14 @@ the MIT License. See LICENSE in the project root for license information. -->
 
     <StackPanel x:Name="HeaderStackPanel"
                 Orientation="Horizontal">
+        <ProgressRing x:Name="HeaderProgressRing"
```

Comment:
Oh, make sure to use the `ProgressRing` from the `Microsoft.UI.Xaml` namespace, not the normal WUX one! The MUX one is the one that has the determinate progress state and the updated visuals, the WUX one does not
![image](https://user-images.githubusercontent.com/18356694/101512497-ae4e7680-3940-11eb-8031-affd7a753b45.png)